encoders/numeric/UMAPEncoder/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM jinaai/jina
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+# install the third-party requirements
+RUN pip install -r requirements.txt
+
+# for testing the image
+RUN pip install pytest && pytest
```

Comment:
see above comment from @rutujasurve94 about removing. The "FROM base" lines were originally auto-generated by the jina hub wizard. I don't fully understand why it is needed at line 11, but am happy to keep it there since it was auto-generated. @rutujasurve94 @nan-wang what do you think? Should these lines stay in or come out?

Reply:
The motivation is to avoid installing pytest and keep the pytest leftovers in the final docker image. When the others use this pod image, they don't need to have these tests related things.

For reference:
- multistage dockerfile: https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
Thanks for the explanation.